client/my-sites/subscribers/style.scss

```diff
@@ -60,7 +60,7 @@
 	background-color: $studio-white;
 }
 
-.is-section-subscribers .layout__content {
+.is-section-subscribers .layout:not( .has-no-sidebar ) .layout__content {
```

Comment:
Small lint issue here:

```
client/my-sites/subscribers/style.scss
   63:37  ✖  Unexpected whitespace after "("   selector-pseudo-class-parentheses-space-inside
   63:53  ✖  Unexpected whitespace before ")"  selector-pseudo-class-parentheses-space-inside
```

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/86556/commits/abfdb707ef7dcf7e2091252d2fe9fee41b943780